### PR TITLE
Setting up a clean environment before automation

### PR DESCRIPTION
**Pull request by elsegev**

### CODE REVIEW

File: .github/workflows/dsf-dam_poc_cli.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: 'DSF & DAM POC CLI'
+name: 'DSF POC CLI - ALL'
```
Comment (lindanasredin) [inline .github/workflows/dsf-dam_poc_cli.yml]: Adding "All" is not aligned with the convention Assaf and I put in place. What does it mean? No other yml has All

Comment (lindanasredin) [reply .github/workflows/dsf-dam_poc_cli.yml]: This file should be renamed to dsf_poc_cli.yml before the next release

File: .github/workflows/dsf-dam_poc_cli.yml [hunk 2]
```diff
@@ -60,7 +60,7 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - name: DSF POC CLI
+          - name: DSF POC CLI - ALL
```
Comment (lindanasredin) [inline .github/workflows/dsf-dam_poc_cli.yml]: All

File: .github/workflows/dsf-dam_poc_cli.yml (hunk 1 above)
Comment (elsegev) [reply .github/workflows/dsf-dam_poc_cli.yml]: Please see again, I removed dsf-dam_poc_cli.yml and used the dsf_poc_cli.yml for it.

Comment (lindanasredin) [reply .github/workflows/dsf-dam_poc_cli.yml]: Great, thanks

**Pull request merged**